### PR TITLE
uk.co.mangobrain.Infector

### PR DESCRIPTION
**Pull request by mangobrain**
Simple Ataxx/Hexxagon game

### CODE REVIEW

File: uk.co.mangobrain.Infector.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	<project_license>GPL-3.0+</project_license>
+    <name>Infector</name>
+    <summary>Play the classic two- to four-player board game Ataxx</summary>
+    <description>
+        <p>
+            Infector is a version of the classic board games Ataxx and
+            Hexxagon. Play alone against simple AI, or with up to four
+            players, locally or over a network.
+        </p>
+    </description>
```
Comment (Alexander-Wilms) [inline uk.co.mangobrain.Infector.appdata.xml]: Some screenshots would be nice

```
<screenshots>
		<screenshot type="default">
			<image type="source">http://flarerpg.org/wp-content/uploads/2018/03/empyrean_1.jpg</image>
                </screenshot>
</screenshots>
```

**Comment by nedrichards** [comment]
bot, build uk.co.mangobrain.Infector

**Comment by Alexander-Wilms** [comment]
All builds succeeded: https://flathub.org/builds/#/builders/2/builds/298

File: uk.co.mangobrain.Infector.appdata.xml (hunk 1 above)
Comment (nedrichards) [reply uk.co.mangobrain.Infector.appdata.xml]: Yeah, we need screenshots. Other than that, looks good to me.

**Comment by nedrichards** [comment]
bot, build uk.co.mangobrain.Infector

**Comment by Alexander-Wilms** [comment]
Failed on arm, but I think that's unrelated to your manifest:
```
FAILED: src/infector@exe/gameboard.cxx.o 
ccache c++  -Isrc/infector@exe -Isrc -I../src -I/app/include/gtkmm-3.0 -I/app/lib/gtkmm-3.0/include -I/app/include/atkmm-1.6 -I/app/include/gdkmm-3.0 -I/app/lib/gdkmm-3.0/include -I/app/include/giomm-2.4 -I/app/lib/giomm-2.4/include -I/app/include/pangomm-1.4 -I/app/lib/pangomm-1.4/include -I/app/include/glibmm-2.4 -I/app/lib/glibmm-2.4/include -I/app/include/cairomm-1.0 -I/app/lib/cairomm-1.0/include -I/app/include/sigc++-2.0 -I/app/lib/sigc++-2.0/include -I/usr/include/gtk-3.0/unix-print -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/pango-1.0 -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -O0 -g -O2 -g -std=c++11 -pthread -MD -MQ 'src/infector@exe/gameboard.cxx.o' -MF 'src/infector@exe/gameboard.cxx.o.d' -o 'src/infector@exe/gameboard.cxx.o' -c ../src/gameboard.cxx
ccache: error: /srv/buildbot/.ccache/ccache.conf: No such file or directory
```

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/uk.co.mangobrain.Infector

**Pull request closed without merging**